specification/containerinstance/resource-manager/Microsoft.ContainerInstance/2017-10-01-preview/containerInstance.json

```diff
@@ -0,0 +1,977 @@
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "description": "The name of the resource group to contain the container group to be created or updated.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "containerGroupName",
+            "in": "path",
+            "description": "The name of the container group to be created or updated.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "containerGroup",
+            "description": "The properties of the container group to be created or updated.",
+            "required": true,
+            "in": "body",
+            "schema": {
+              "$ref": "#/definitions/ContainerGroup"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/ContainerGroup"
+            }
+          },
+          "201": {
+            "description": "Created - the container group is created.",
+            "schema": {
+              "$ref": "#/definitions/ContainerGroup"
+            }
+          }
+        }
+      },
+      "delete": {
+        "operationId": "ContainerGroups_Delete",
+        "x-ms-examples": {
+          "ContainerGroupsDelete": {
+            "$ref": "./examples/ContainerGroupsDelete.json"
+          }
+        },
+        "summary": "Delete the specified container group.",
+        "description": "Delete the specified container group in the specified subscription and resource group. The operation does not delete other resources provided by the user, such as volumes.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "description": "The name of the resource group that contains the container group.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "containerGroupName",
+            "in": "path",
+            "description": "The name of the container group to be deleted.",
+            "required": true,
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/ContainerGroup"
+            }
+          },
+          "204": {
+            "description": "No Content - the specified container group was not found."
+          }
+        }
+      }
+    },
+    "/providers/Microsoft.ContainerInstance/operations": {
+      "get": {
+        "operationId": "Operation_List",
```

Comment:
>"operationId": "Operation_List", [](start = 8, length = 32)

Is this a pageable operation?

Reply:
No. We have limited operations at this point? Is it required to be a pageable operation?

Reply:
It makes me think it a [pageable](https://github.com/Azure/autorest/blob/master/docs/extensions/readme.md#x-ms-pageable) operation because `OperationListResult` defines a `nextLink` property.

Reply:
Still needs to get marked as pageable
``` json
"x-ms-pageable": {
      "nextLinkName": null
}
```